src/index.ts

```diff
@@ -555,11 +606,17 @@ export class Bigtable {
    *   const apiResponse = data[2];
    * });
    */
-  createInstance(id, options, callback) {
-    if (is.function(options)) {
-      callback = options;
-      options = {};
-    }
```

Comment:
Document about `createInstance` describe that `option` is not optional but worried with this check.